core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -315,15 +316,30 @@ class UserService(
   def mapCloudPlatformAndPolicies(
     project: RawlsBillingProject,
     billingProfile: Option[ProfileModel],
-    roles: Set[ProjectRole]
+    roles: Set[ProjectRole],
+    workspaceManagerDao: WorkspaceManagerDAO
   ): RawlsBillingProjectResponse = (project.billingProfileId, billingProfile) match {
     case (None, _) => RawlsBillingProjectResponse(roles, project, CloudPlatform.GCP, protectedData = None)
     case (Some(_), Some(p)) =>
       val platform = CloudPlatform(p)
-      val responseProject = if (platform == CloudPlatform.AZURE) {
+      val (responseProject, maybeRegion) = if (platform == CloudPlatform.AZURE) {
         val c = AzureManagedAppCoordinates(p.getTenantId, p.getSubscriptionId, p.getManagedResourceGroupId)
-        project.copy(azureManagedAppCoordinates = Some(c))
-      } else project
+        val projectWithCoords = project.copy(azureManagedAppCoordinates = Some(c))
+        val landingZoneRegion = project.landingZoneId match {
+          case Some(landingZoneId) =>
+            Try(workspaceManagerDao.getLandingZone(UUID.fromString(landingZoneId), ctx)) match {
+              case Success(landingZone) => Option(landingZone.getRegion)
+              case Failure(exception) =>
+                logger.debug(
```

Comment:
Do we want this exceptional condition logged at a higher level?

Reply:
I don't know how exceptional it is… I have a lot of billing projects in the state where the landing zone no longer can be retrieved due to deletion errors. I don't think we want to be creating noise every time billing projects are retrieved due to this.

Furthermore, the UI code will either (pending discussion with Adam) show no warning if the region can't be obtained from the Azure billing project, or display a generic warning about possible egress charges like we do in the case of a requester pays GCP workspace (since we can't get the bucket location there either).

Reply:
I feel like if there is a fundamental problem with retrieving landing zones, we will end up erroring/alerting other places which are more in a critical path than this one, which is simply informational.

Reply:
Good context. Thanks!